inclusive-design/idrc
# feat: replace custom filters with eleventy-plugin-fluid

## Description

Replaces custom filter implementation with [eleventy-plugin-fluid](https://github.com/fluid-project/eleventy-plugin-fluid).

## Steps to test

1. Run build.

**Expected behavior:** Site builds with filters applied correctly.

## Additional information

Not applicable.

## Related issues

Not applicable.
